.pre-commit-config.yaml

```diff
@@ -23,3 +24,7 @@ repos:
         language: system
         files: \.(js|vue|scss|css)$
         exclude: '^kolibri/core/static/assets/fonts/.+?\.css$|^kolibri/core/content/static/assets/'
+-   repo: https://github.com/psf/black
+    rev: stable
+    hooks:
+    - id: black
```

Comment:
For Python2.7 development, this probably needs to specify 

```
    hooks:
      - id: black
        language_version: python3.6
```

However, the main issue is that it still requires Python >= 3.6. See previous conversation here:  https://github.com/psf/black/issues/585#issuecomment-478365434

This is what led to the creation of https://github.com/learningequality/black-fmt

Reply:
Ah, looks like we already had this conversation: https://github.com/learningequality/black-fmt/issues/3

Reply:
Hm I forgot you made this poll

![image](https://user-images.githubusercontent.com/2367265/75081461-21772100-54c4-11ea-8538-03f9cdbb297b.png)

So perhaps we're now in a position to require >= 3.6 after all!

Reply:
I think it doesn't need be specified. It will install black for the provided python version. pre-commit creates its own virtual environment for the hooks and runs them independently from the external (virtual) environment.

I just tried it from a Python 2.7 environment, and a Python 3 environment was created instead. 

https://pre-commit.com/#overriding-language-version